src/helpers/storageHelper.js

```diff
@@ -17,7 +17,9 @@ export const storageHelper = {
   listTypesSettings: () => readFromStore('listTypesSettings'),
   setListTypesSettings: (listTypesSettings) => addToStore('listTypesSettings', listTypesSettings),
   configurations: () => readFromStore('configurations'),
-  setConfigurations: (configurations) => addToStore('configurations', configurations)
+  setConfigurations: (configurations) => addToStore('configurations', configurations),
+  filter: () => readFromStore('filter'),
+  setFilter: (filter) => addToStore('filter', filter)
```

Comment:
resource...

Reply:
name change implemented with this commit
https://github.com/smart-village-solutions/smart-village-app-app/pull/1188/commits/6832d3814e0fced2f8c3613d1e569788bdd5398a